presto-iceberg/src/test/java/io/prestosql/plugin/iceberg/TestIcebergSmoke.java

```diff
@@ -248,30 +247,31 @@ private void testCreatePartitionedTable(Session session, FileFormat fileFormat)
                 ", CAST('2017-05-01 10:12:34' AS TIMESTAMP) _timestamp" +
                 ", CAST('2017-05-01' AS DATE) _date";
 
-        assertUpdate(session, "INSERT INTO test_partitioned_table " + select, 1);
-        assertQuery(session, "SELECT * from test_partitioned_table", select);
-        assertQuery(session, "" +
-                        "SELECT * FROM test_partitioned_table WHERE" +
-                        " 'foo' = _string" +
-                        " AND 456 = _integer" +
-                        " AND CAST(123 AS BIGINT) = _bigint" +
-                        " AND true = _boolean" +
-                        " AND CAST('3.14' AS DECIMAL(3,2)) = _decimal_short" +
-                        " AND CAST('12345678901234567890.0123456789' AS DECIMAL(30,10)) = _decimal_long" +
-                        " AND CAST('2017-05-01 10:12:34' AS TIMESTAMP) = _timestamp" +
-                        " AND CAST('2017-05-01' AS DATE) = _date",
-                select);
+        assertUpdate(format("INSERT INTO test_partitioned_table %s", select), 1);
+        assertQuery("SELECT * FROM test_partitioned_table", select);
 
-        dropTable(session, "test_partitioned_table");
+        String selectAgain = "" +
+                "SELECT * FROM test_partitioned_table WHERE" +
+                " 'foo' = _string" +
+                " AND 456 = _integer" +
+                " AND CAST(123 AS BIGINT) = _bigint" +
+                " AND true = _boolean" +
+                " AND CAST('3.14' AS DECIMAL(3,2)) = _decimal_short" +
+                " AND CAST('12345678901234567890.0123456789' AS DECIMAL(30,10)) = _decimal_long" +
+                " AND CAST('2017-05-01 10:12:34' AS TIMESTAMP) = _timestamp" +
+                " AND CAST('2017-05-01' AS DATE) = _date";
+        assertQuery(selectAgain, select);
+
+        dropTable("test_partitioned_table");
     }
 
     @Test
     public void testCreatePartitionedTableWithNestedTypes()
     {
-        testWithAllFileFormats(this::testCreatePartitionedTableWithNestedTypes);
+        testWithAllFileFormats(this::testCreatePartitionedTableWithNestedTypesForFormat);
```

Comment:
instead doing loop manually you can use `@DataProvider`
Example: https://github.com/prestosql/presto/blob/93faf7ad9a6b95a2436d4eaa9bf567756b759c3f/presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveStorageFormats.java#L63

Reply:
Thanks for the tip; I'll take a look.  Today, the Iceberg connector supports `ORC` and `Parquet`, and there is a PR to support `AVRO`.

`TestIcebergSmoke` is part of module `presto-iceberg`.  I was under the impression that we could not introduce dependencies on `presto-product-tests` in modules like `presto-iceberg`.